frontend/app/components/wp-edit-form/work-package-edit-context.ts

```diff
@@ -1,12 +1,12 @@
-//-- copyright
+// -- copyright
 // OpenProject is a project management system.
-// Copyright (C) 2012-2017 the OpenProject Foundation (OPF)
+// Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
```

Comment:
That's caused by dev having updated the copyright notices in the meantime. Fixed